Update structure of project

MikhailJunker opened the pull request:
This pull request contain 4 commits, which: 
- Deleted configuration files of programming language and IDE
- Deleted some excesses source files, modify all remaining source files(for more convenient output of the work result) and added comments at code
- Update .gitignore file
- Change folder's structure of project

rrtty0 committed:
Update README.md

MikhailJunker committed:
Change structure of Project

Delete configuration files of programming language
and IDE, renamed source files and changed the
structure of project

MikhailJunker committed:
Created .gitignore

Created ignore file and filled it

MikhailJunker committed:
Rechanged structure of project

Deleted some excesses source files,
modify all remaining source files and
added comments at code

rrtty0 commented on source/HashTable.cpp:
```diff
@@ -1,10 +1,9 @@
-
+//constructor without arguments
 HashTable::HashTable()
 {
```
Please, move the brackets to line with function declaration for all functions

rrtty0 commented:
Good job! I like this changes, but i write some comment about code at file HashTable.cpp, which needs fixing

MikhailJunker committed:
Update file HashTable.cpp

Moved start bracket of every function to line with function declaration
for all functions at this file

MikhailJunker commented:
Done a new commit with the necessary changes

rrtty0 commented:
Ok, thank you for your job! I will merge your changes at my repository.
